components/Pagination.js

```diff
@@ -112,7 +113,7 @@ class Pagination extends Component {
       ...other
     } = this.props;
 
-    const { page, pageSize } = this.state;
+    const { sPage, sPageSize } = this.state;
```

Comment:
Was it with this line that made the prop and state naming confusing? 
Instead of renaming our `state` keys, I would prefer that we reference state within this `render()` method as `this.state.page` or `this.state.pageSize`

cc: @iangfleming @tw15egan @chrisdhanaraj @marijohannessen